### PR TITLE
Address mangement

### PR DESCRIPTION
**Pull request by AnuragSahu11**
*Added Address management
*Added Profile page
*Added Order history, available in profile page
*Refactored old code
*Added Address Controller and updated server.js file according to the address controller

### CODE REVIEW

File: lucid-book-store/src/pages/user-profile/user-profile-page.jsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import { Link } from "react-router-dom";
+import { OrderCard } from "../../components/cards/order-card";
+import { useData } from "../../context";
+
+const UserProfilePage = () => {
+  const {
+    dataState: { orders },
+  } = useData();
+  console.log(orders);
```
Comment (junaidshaikh-js) [inline lucid-book-store/src/pages/user-profile/user-profile-page.jsx:9]: remove console logs

**Review by junaidshaikh-js — APPROVED** [review]
Good work! 

Things are working as expected. PR can be merged. 

Try to divide tasks into smaller PRs

**Review by saadahmadhere — APPROVED** [review]
Code looks clean after refactoring.
- Modular code.
- Nice use of css variables.
- Good to see the 'search' feature.

File: lucid-book-store/src/pages/user-profile/user-profile-page.jsx (hunk 1 above)
Comment (AnuragSahu11) [reply lucid-book-store/src/pages/user-profile/user-profile-page.jsx:9]: removed

**Pull request merged**